zam/aspirateur/zam_aspirateur/amendements/parser.py

```diff
@@ -0,0 +1,51 @@
+from datetime import datetime
+
+from ..clean import clean_html
+
+from .models import Amendement
+
+
+def parse_from_csv(d_amend: dict) -> Amendement:
```

Comment:
I'm not sure it's parsing from csv anymore? Maybe `Amendement.from_csv` and `.from_json`?

Reply:
Hmm, the `parse_from_csv` name is not great... Its not really parsing a CSV file, its more building an Amendement instance based on a CSV row. Maybe `build_from_csv_row` then?